### PR TITLE
DO NOT MERGE. Passport fix.

### PR DESCRIPTION
**Pull request by rafkhan**
@bennett000 
@SethDavenport when this issue is resolved (and tested), the runtime should be good to go.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
   "dependencies": {
     "aws-sdk": "^2.3.11",
     "base64url": "^1.0.5",
-    "body-parser": "^1.15.1",
+    "body-parser": "^1.14.2",
```
Comment (bennett000) [inline package.json]: This was not the problem we should stick to `1.15.1`

File: src/server/clusternator-api/api-0.1.js [hunk 2]
```diff
@@ -133,9 +139,13 @@ function init(app, projectDb) {
...
+
+  console.log('authC is a function?', typeof authC, authC);
+
   app.post(`/${API}/:namespace/:command`, [
-    authHeaderHandler,
-    authorizeCommand(config),
+    passport.authenticate(['auth-header']),
+    authC,
     executeCommand(commands)
   ]);
```
Comment (bennett000) [inline src/server/clusternator-api/api-0.1.js]: The console logs will need to be removed

File: src/server/main.js [hunk 3]
```diff
@@ -59,6 +59,10 @@ function initExpress(app) {
 
   app.use(bodyParser.json());
   app.use(loggers.request);
+  app.use(function (err, req, res, next) {
+    console.log('GLOBAL ERROR', err);
```
Comment (bennett000) [inline src/server/main.js]: another console log

**Comment by bennett000** [comment]
:shipit:

**Comment by SethDavenport** [comment]
Awesome, I really want the runtime to be working :) Then I can move the starters off Heroku.

**Pull request merged**